api/envoy/extensions/filters/http/mcp/v3/mcp.proto

```diff
@@ -17,3 +18,8 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message Mcp {
   // TODO: Add configuration fields
 }
+
+// Per-route configuration for MCP filter
+message McpPerRoute {
```

Comment:
Do we expect that there will be a different set of fields needed in the override config than in the top-level config?  If not, we could use the same proto for both.

If we don't really know what we'll need yet and are using a different proto name here just in case they need to diverge, I'm okay with that.  I just want to make sure we've at least considered the question.